### DISCUSSION TITLE
Developer Mode for Jagex Launcher

### DISCUSSION THREAD

**Discussion by cbartram** [runelite/runelite#16492] [Ideas]
Hello, 

I regularly compile and run runelite from source code so that I can enable the developer tools. I use the wall and object information to export models that I edit in blender for my YouTube videos. Since I migrated my accounts to Jagex's new account system I can no longer login to RuneLite if it is built from source. I can only login to RuneLite if it was launched via the Jagex Launcher.

This wouldn't be a problem except for the fact that I need to run the client in developer mode. It's as simple as passing in a CLI argument but I don't think I can do that through Jagex's launcher. Any thoughts around this as more legacy accounts are migrated to "Jagex Accounts"?

**Comment by MESLewis** [comment]
There is no way to use developer mode with the Jagex launcher. There is a model exporter available on the plugin hub: https://runelite.net/plugin-hub/show/model-exporter

**Comment by cbartram** [comment]
The model exporter unfortunetly cannot export any models that are not examinable which is most objects that make up a scene. i.e dirt, tiles, walls, objects outside playable game area etc...

**Comment by Adam-** [comment]
We aren't interested in this due to the potential abuse.
